openhtf/plugs/README.md

```diff
@@ -0,0 +1,10 @@
+# Usefull thinks!
+## Template to create Plugs
+* (Plug Template)[https://github.com/jo-nas/openhtf_plug_template]: A cookiecutter template for OpenHTF plugs.
+
+# List of Plugins
```

Comment:
s/Plugins/Plugs

I would re-word this as `List of Known Third-Party Plugs` for greater specificity.